### PR TITLE
espanso-alpha: Add version v2.1.3-alpha

### PR DESCRIPTION
**Pull request by hmnd**
Adds alpha version of [espanso](https://espanso.org/)

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Use the portable version

File: bucket/espanso-alpha.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "innosetup": true,
+    "bin": [
+        "espansod.exe",
+        "espanso.cmd"
+    ],
+    "shortcuts": [
+        [
+            "espansod.exe",
+            "Espanso",
+            "start",
```
Comment (rashil2000) [inline bucket/espanso-alpha.json:24]:
```suggestion
            "launcher",
```

Shouldn't this be "launcher" instead?

**Comment by rashil2000** [comment]
> Use the portable version

**Review by tech189 — CHANGES_REQUESTED** [review]
Take a look at my suggestions - they change it from an installer to portable, fix autoupdate and add the icon which is missing from portable version :)

**Comment by rashil2000** [comment]
@tech189 can you open a new superseding PR with your suggestions?

**Pull request closed without merging**